.github/workflows/release-connect-bump-versions.yml

```diff
@@ -23,6 +23,12 @@ jobs:
         uses: actions/checkout@v4
         with:
           token: ${{ secrets.TREZOR_BOT_TOKEN }}
+          # Fetch all commit history because Script connect-bump-versions.ts needs them to produce CHANGELOG for packages.
+          fetch-depth: 0
+          # `ref` makes sure that we checkout
```

Comment:
makes sure that we checkout <something>? is the sentence finished ? : P

Reply:
Ups, you are right, I just completed the sentence. Thanks!